cmd/nerdctl/container/container_remove_test.go

```diff
@@ -24,16 +24,18 @@ import (
 
 func TestRemoveContainer(t *testing.T) {
 	t.Parallel()
+
 	base := testutil.NewBase(t)
 	tID := testutil.Identifier(t)
 
 	// ignore error
 	base.Cmd("rm", tID, "-f").AssertOK()
 
-	base.Cmd("run", "-d", "--name", tID, testutil.CommonImage, "sleep", "infinity").AssertOK()
+	base.Cmd("run", "-d", "--name", tID, testutil.NginxAlpineImage).AssertOK()
```

Comment:
?

Reply:
The same test was duplicated for Windows and Linux (now merged).
The windows version was using `NginxAlpineImage` - the linux version was using `CommonImage`.
I picked the windows version.